include/common_structs.h

```diff
@@ -354,15 +358,13 @@ typedef struct Evt {
     /* 0x070 */ union {
     /*       */     s32 functionTemp[4];
     /*       */     f32 functionTempF[4];
-    /*       */     struct Npc* functionTempNpc[4];
-    /*       */     struct Actor* functionTempActor[4];
-    /*       */     struct ActorPart* functionTempActorPart[4];
+    /*       */     void* functionTempPtr[4];
```

Comment:
I think we may want to keep the commonly-used types as separate fields for the sake of documenting the code? though I don't feel super strongly about that..

Reply:
I'm happy with just void* personally